kraken-std/src/kraken/std/cargo/tasks/cargo_generate_deb.py

```diff
@@ -0,0 +1,43 @@
+import subprocess
+from dataclasses import dataclass
+from pathlib import Path
+
+from kraken.core import Property, Task, TaskStatus
+
+from kraken.std.descriptors.resource import LibraryArtifact
+
+
+@dataclass
+class CargoDebianArtifact(LibraryArtifact):
+    pass
+
+
+class CargoGenerateDebPackage(Task):
+    package_name: Property[str] = Property.default("CargoDebian")
+    skip: Property[bool] = Property.default(False)
```

Comment:
Why do we need a skip property? If the caller is able to determine they do not want the task to run when creating it, they might as well not create it, no?